content/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect.md

```diff
@@ -466,6 +466,12 @@ To configure the repository to use the organization's template, a repository adm
 You can now update your YAML workflows to use OIDC access tokens instead of secrets. Popular cloud providers have published their official login actions that make it easy for you to get started with OIDC. For more information about updating your workflows, see the cloud-specific guides listed below in "[Enabling OpenID Connect for your cloud provider](#enabling-openid-connect-for-your-cloud-provider)."
 
 
+## Enabling OpenID Connect for Python package publishing
+
+The [Python Package Index (PyPI)](https://pypi.org) supports "token exchange" with OpenID connect: you can configure a GitHub workflow in a repository as a trusted publisher for a PyPI project, allowing OIDC access tokens to be exchanged for temporary PyPI API tokens.
+
+See the guide in "[AUTOTITLE](/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-pypi)" for more information, as well as PyPI's [own documentation on trusted publishing](https://docs.pypi.org/trusted-publishers/).
```

Comment:
Went ahead and updated this to `docs.pypi.org`, since the docs there are more detailed than the basic "help" page that was linked before.